libs/utils/android/system.py

```diff
@@ -47,8 +47,12 @@ def systrace_start(target, trace_file, time=None,
                 return None
 
         #  Format the command according to the specified arguments
-        systrace_pattern = "{} -e {} -o {} {}"
-        trace_cmd = systrace_pattern.format(systrace_path, target.conf['device'],
+        try:
+            device = "-e {}".format(target.conf['device'])
+        except:
+            device = ''
```

Comment:
It's not about number of lines... point is that, most of the times 'device' is in `target.conf` and that version just skip the test. It's considered to be (unnoticeable) faster and more "python" ;-)